src/diagram/components/quantity-node.tsx

```diff
@@ -107,37 +126,34 @@ const _QuantityNode: React.FC<IProps> = ({ data, isConnectable }) => {
             </div>
           </div>
         }
-        <div className="variable-info-row">
-          <input className="variable-info value" type="number" placeholder="value" autoComplete="off" onChange={onValueChange} data-testid="variable-value"
-            value={shownValue !== undefined ? shownValue.toString() : ""} onMouseDown={e => e.stopPropagation()} />
-          <input className="variable-info unit" type="text" placeholder="unit" autoComplete="off" value={shownUnit|| ""} data-testid="variable-unit"
-            onChange={onUnitChange} onMouseDown={e => e.stopPropagation()} />
-        </div>
-        <div>
-          <select className="variable-info operation" value={data.node.variable.operation || ""} onChange={onOperationChange}>
-          { // in an enumeration the keys are the names and the values are string or numeric identifier
-          }
-          <option key="none" value="">none</option>
-          {Object.entries(Operation).map(([name, symbol]) =>
-            <option key={name} value={symbol} data-testid={`variable-operation-${name}`}>{symbol}</option>
-          )}
-          </select>
-          { variable.computedValueError &&
-            <div className="error-message">
-                ⚠️ {variable.computedValueError}
-            </div>
-          }
-          { variable.computedUnitError &&
-            <div className="error-message">
-                ⚠️ {variable.computedUnitError}
-            </div>
-          }
-          { variable.computedUnitMessage &&
-            <div className="error-message">
-                ⓘ {variable.computedUnitMessage}
-            </div>
-          }
-        </div>
+        {variable.numberOfInputs >= 1 ? renderValueUnitUnEditable() : renderValueUnitInput()}
+        {!(variable.numberOfInputs >= 1) &&
```

Comment:
I don't understand this. If I'm reading it right, it means the error messages will not be shown if there are inputs. But the error messages are most important when there are inputs.